dags.yaml

```diff
@@ -240,3 +240,12 @@ bqetl_org_mozilla_fenix_derived:
     retry_delay: 30m
     start_date: '2020-10-18'
   schedule_interval: daily
+
+bqetl_google_analytics_derived:
+  schedule_interval: 0 16 * * *
```

Comment:
Can we comment here on why this runs at 16:00 UTC? I assume it has to do with the schedule for GA delivery to BQ.

Reply:
Yes the GA export runs at 15:00 UTC.  The existing derived table jobs run at 16:00 and there haven't been issues so I'll continue that here.  I'll add a comment.